src/ArtDetails/ArtDetails.tsx

```diff
@@ -25,7 +25,7 @@ interface Details {
 
```

Comment:
For your interfaces in this file (and others), you might want to make an additional "types" file to store your interfaces for organization.